Cargo.toml

```diff
@@ -1,8 +1,8 @@
 [workspace]
-members = [
-    "meilisearch-http",
-    "meilisearch-error",
-]
+members = ["meilisearch-http", "meilisearch-error"]
 
 [profile.release]
 debug = true
+
+[patch.crates-io]
+pest = { git = "https://github.com/pest-parser/pest.git", rev = "51fd1d49f1041f7839975664ef71fe15c7dcaf67" }
```

Comment:
this is the main change of the PR.
The Cargo.toml has just been recreated to take this change into account.